docs/openapi/components/schemas/common/TradeLineItem.yml

```diff
@@ -70,17 +70,17 @@ properties:
     description: A measure of the net weight (mass).
     $ref: ./QuantitativeValue.yml
     $linkedData:
-      term: netWeight
+      term: tradeLineItemNetWeight
```

Comment:
is there a reason this and other similar terms like `tradeLineItemGrossWeight` are prefixed by `tradeLineItem`?

Why isn't this just `netWeight` or `grossWeight`?

Reply:
This should be fixed... prefxing is against the JSON-LD conventions we have.

Reply:
Agreed, @mkhraisha , let's stick to the no-prefix convention.